File: ilias-connector/ilias-connector-api/src/main/java/de/adesso/iliasdownloader3/service/exception/IliasAuthenticationException.java
package de.adesso.iliasdownloader3.service.exception;

public final class IliasAuthenticationException extends IliasException {

    private static final long serialVersionUID = 965016465165430L;

    public IliasAuthenticationException(String message) {
        super(message);
    }

    public IliasAuthenticationException(String message, Throwable t) {
        super(message, t);
    }

    public IliasAuthenticationException(Throwable t) {
        super(t);
    }
}

File: src/main/kotlin/com/github/thetric/iliasdownloader/connector/api/exception/IliasAuthenticationException.kt
package com.github.thetric.iliasdownloader.connector.api.exception

/**
 * Authentication at the Ilias web service failed.
 */
class IliasAuthenticationException : IliasException {
    constructor(message: String) : super(message)

    constructor(message: String, t: Throwable) : super(message, t)

    constructor(t: Throwable) : super(t)
}
